### ISSUE TITLE
Searches for Movies aren't returning Year in the results at all times

### ISSUE THREAD

**Issue by almightiest** [cinemagoer/cinemagoer#453]
#### Issue description

When searching for "Return of the Jedi" I would expect the results to properly parse the year 1983, but instead the list of Movie objects mostly contain a None year.

#### Version of Cinemagoer, Python and OS

- **Python:** 3.10.6
- **Cinemagoer:** 2022.12.27
- **OS:** uname_result(system='Linux', node='media', release='5.19.0-38-generic', version='#39~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Mar 17 21:16:15 UTC 2', machine='x86_64')

#### Steps to reproduce the issue

*if possible, provide a minimal code to reproduce the problem*

```
$ search_movie.py "return of the jedi"
    20 results for "return of the jedi":
movieID : imdbID : title
0086190 : 0086190 : Star Wars: Episode VI - Return of the Jedi (None)
8925010 : 8925010 : Star Wars: Episode VI - Return of the Jedi: Deleted Scenes (None)
12516558        : 12516558 : Star Wars: Return of the Jedi - The Original Radio Drama (None)
19392076        : 19392076 : Return of The Jedi: A Bad Lip Reading (None)
2937992 : 2937992 : The Return of Return of the Jedi: 30 Years and Counting (None)
0400304 : 0400304 : Classic Creatures: Return of the Jedi (None)
0185712 : 0185712 : Super Star Wars: Return of the Jedi (None)
3169422 : 3169422 : Return of the Jedi 30th Anniversary Celebration at the Egyptian Theatre (None)
2301123 : 2301123 : Return of the Jedi (None)
7403800 : 7403800 : Super Star Wars: Return of the Jedi (None)
0299155 : 0299155 : Star Wars: Return of the Jedi - Death Star Battle (None)
2013783 : 2013783 : Return of the Jedi (None)
7237678 : 7237678 : Return of the Jedi (None)
24552314        : 24552314 : Return of the Jedi (None)
26356882        : 26356882 : Return of the Jedi (None)
16799060        : 16799060 : Return of the Jedi (None)
9734618 : 9734618 : Return of the Jedi (None)
24553134        : 24553134 : Return of the Jedi (None)
24819462        : 24819462 : Return of the Jedi (None)
7261152 : 7261152 : Return of the Return of the Jedi (None)
```

```
$ get_movie.py 0086190
Movie
=====
Title: Star Wars: Episode VI - Return of the Jedi (1983)
Genres: Action, Adventure, Fantasy, Sci-Fi.
Director: Richard Marquand.
Writer: Lawrence Kasdan, George Lucas, , George Lucas.
Cast: Mark Hamill (Luke Skywalker), Harrison Ford (Han Solo), Carrie Fisher (Princess Leia), Billy Dee Williams (Lando Calrissian), Anthony Daniels (C-3PO).
Runtime: 131.
Country: United States.
Language: English, Kikuyu, Tagalog, Kalmyk-Oirat, Quechua, Polish.
Rating: 8.3 (1070800 votes).
```

#### What's the expected result?

- Search results for 0086190 movie id should have 1983 as the year

#### What's the actual result?

- Search results for 0086190 movie id has None as the year

#### Additional details

- I used the provided search_movie.py script to make it easy to reproduce.  Having the year in the initial list without having to query deeper into each result makes movie matching much more efficient.

**Comment by h3llrais3r** [comment]
Facing the same issue.

**Comment by mumixam** [comment]
this is fixed via https://github.com/cinemagoer/cinemagoer/commit/23c4bec1e41c260244b7f6dafcf46012a316790e
the verison on pypi/pip is older than this fix.
try using ```pip install git+https://github.com/cinemagoer/cinemagoer``` or maunally update the imdb/parser/http/searchMovieParser.py to the current from the master branch

**Comment by almightiest** [comment]
> this is fixed via [23c4bec](https://github.com/cinemagoer/cinemagoer/commit/23c4bec1e41c260244b7f6dafcf46012a316790e) the verison on pypi/pip is older than this fix. try using `pip install git+https://github.com/cinemagoer/cinemagoer` or maunally update the imdb/parser/http/searchMovieParser.py to the current from the master branch

That worked, thanks!
